ch_3_AppDev_Basic/3_Baichuan2_BasicApp.ipynb

```diff
@@ -28,6 +28,15 @@
     "!pip install --pre --upgrade ipex-llm[all]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> **Note**\n",
+    ">\n",
+    "> Please refer to https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_cpu.html#quick-installation for more details on how to set up the environment."
```

Comment:
It maybe better to remind users that: 

"On Linux OS,  we recommend to use `pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu` to install. Please refer to https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_cpu.html#quick-installation for more details."